app/views/vacancies/index.html.haml

```diff
@@ -8,7 +8,7 @@
     = render 'filters'
   .govuk-grid-column-two-thirds
     .govuk-grid-column-full.vacancies-count{class: @vacancies.user_search? ? 'search-made' : ''}
-      %p.govuk-heading-l.mt0.mb1.inline-block#vacancy-results= @vacancies.total_count_message
+      %p.govuk-heading-l.mt0.mb1.inline-block#vacancy-results= @filters.location_category_search? ? @vacancies.total_count_message_with_location(@filters.location) : @vacancies.total_count_message
```

Comment:
This is identical to the change in `index.html+phone.haml`, moving this logic into `VacanciesPresenter` could reduce duplication